src/course-unit/breadcrumbs/Breadcrumbs.jsx

```diff
@@ -14,6 +17,17 @@ const Breadcrumbs = () => {
   const intl = useIntl();
   const { ancestorXblocks } = useSelector(getCourseSectionVertical);
   const [section, subsection] = ancestorXblocks ?? [];
+  const navigate = useNavigate();
+  const waffleFlags = useSelector(getWaffleFlags);
+
+  const handleClick = (e, url) => {
+    e.preventDefault();
+    if (waffleFlags?.useNewCourseOutlinePage) {
+      navigate(url);
+    } else {
+      window.location.href = `${getConfig().STUDIO_BASE_URL}${url}`;
+    }
+  };
```

Comment:
Cann this also just be `<Link to={waffleFlags?.useNewCourseOutlinePage ? url : `${getConfig().STUDIO_BASE_URL}${url}`}>...` ?

Reply:
Implemented in follow-up PR: [`885ecab` (#1372)](https://github.com/openedx/frontend-app-authoring/pull/1372/commits/885ecaba6852f91719b84a62ea9b660c423b25ee)